src/dataset/composer.py

```diff
@@ -34,3 +34,45 @@ def __call__(self, image: np.ndarray, annotations: Dict) -> Any:
             image, annotations = func(image, annotations, **self.kwargs)
 
         return image, annotations
+
+
+def join_patches(patches: List[np.ndarray], filenames: List[str]) -> np.ndarray:
+    if len(patches) == 0:
+        raise ValueError("Input list of patches is empty.")
+
+    patches_positions = [x_y_from_filename(filename) for filename in filenames]
+    result_shape = calculate_result_shape_from_patches(patches, patches_positions)
+    result_image = np.zeros(result_shape, dtype=np.uint8)
+
+    for (y_at_result_image, x_at_result_image), patch in zip(patches_positions, patches):
```

Comment:
Suggestion: improve variable names. e.g. (init_y, init_x), (row, col), (y, x)...

Reply:
I just saw that you pushed a commit called "[Better names for variables](https://github.com/laradru/ai-palatal-rugoscopy/pull/11/commits/f53cf25b63debd40244e91bc249a59eda8af0018)" :rofl:

Reply:
Well, let me know if you're ok with the new names.